### ISSUE TITLE
Rename `Velocity` to `Speed`

### ISSUE THREAD

**Issue by kemenaran** [zladx/LADX-Disassembly#369]
Most speed-related variables are named `Speed`-Something – but some still use the outdated `Velocity` name.

We should rename variables and comments so that “speed” is used everywhere.

**Comment by kemenaran** [comment]
Hmm, interesting. Like speed is a fixed attribute of an entity (how fast can it move?), and velocity is “how fast does it moves right now”?

In that case making the variables consistent probably means renaming them to `Velocity` rather than `Speed`.

**Comment by daid** [comment]
> Hmm, interesting. Like speed is a fixed attribute of an entity (how fast can it move?), and velocity is “how fast does it moves right now”?

Yes, that's how I usually use it. But other people will most likely disagree, we are on the internet after all :D

**Comment by ealvesss** [comment]
Can I Get This Issue?

**Comment by daid** [comment]
You can do what you want, we dont work as a strict organization here.
